packages/common/src/store/collectibles/reducer.ts

```diff
@@ -1,30 +1,65 @@
 import { createReducer, ActionType } from 'typesafe-actions'
 
-import { Collectible, ID } from 'models'
+import { Chain, Collectible, ID } from '../../models'
 
 import * as actions from './actions'
-import { UPDATE_USER_COLLECTIBLES } from './actions'
+import { UPDATE_USER_ETH_COLLECTIBLES, UPDATE_USER_SOL_COLLECTIBLES, UPDATE_SOL_COLLECTIONS } from './actions'
+import { Metadata } from '@metaplex-foundation/mpl-token-metadata'
+import { Nullable } from '../../utils'
 
 type CollectiblesActions = ActionType<typeof actions>
 
-type CollectiblesState = {
-  collectibles: { [id: ID]: Collectible[] }
+export type CollectiblesState = {
+  userCollectibles: {
+    [id: ID]: {
+      [Chain.Eth]: Collectible[],
+      [Chain.Sol]: Collectible[]
+    }
+  },
+  solCollections: {
+    [mint: string]: (Metadata & { imageUrl: Nullable<string> })
+  }
 }
 
 const initialState = {
-  collectibles: {}
+  userCollectibles: {},
+  solCollections: {}
 }
 
 const collectiblesReducer = createReducer<
```

Comment:
from @piazzatron 
"why are we using createReducer instead of redux-toolkit create-slice here?"

Reply:
updated to use slice